tests/functional/aws-node-sdk/test/object/putObjAcl.js

```diff
@@ -65,23 +64,18 @@ describe('PUT Object ACL', () => {
             return bucketUtil.deleteOne(bucketName);
         });
 
-        it('should put object ACLs', done => {
+        it('should put object ACLs', async () => {
             const s3 = bucketUtil.s3;
             const Bucket = bucketName;
             const objects = [
                 { Bucket, Key },
             ];
-
-            Promise
-                .mapSeries(objects, param => s3.putObject(param).promise())
-                .then(() => s3.putObjectAcl({ Bucket, Key,
-                    ACL: 'public-read' }).promise())
-                .then(data => {
-                    assert(data);
-                    done();
-                })
-                .catch(done);
-        });
+            for (const param of objects) {
+                await s3.putObject(param).promise();
+            }
+            const data = await s3.putObjectAcl({ Bucket, Key, ACL: 'public-read' }).promise();
```

Comment:
used to be done in parallel (could still be done with `Promise.all()` or `Promise.allSettled()`) : is it important? may affect the perf/duration of the test?

Reply:
It was done sequentially, as bluebird's mapSeries function was used, and it takes an iterator as input, so that won't change anything